cruise-control/src/yaml/responses/operationProgress.yaml

```diff
@@ -0,0 +1,37 @@
+OperationProgress:
```

Comment:
This file seems redundant, we already have [progressResult.yaml](https://github.com/linkedin/cruise-control/blob/c64d73c3c133dcdd705488b9f42ea5e10d34e3bf/cruise-control/src/yaml/responses/progressResult.yaml#L1)

Reply:
Good point. Quick clarification question for this - is this comment about `StepProgress` or `OperationProgress` in general? JSON structures for [progressResult](https://github.com/TeamCtrlC/cruise-control/blob/master/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/response/ProgressResult.java#L36) and [operationProgress](https://github.com/TeamCtrlC/cruise-control/blob/master/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/async/progress/OperationProgress.java#L107) seems to be a bit different from each other.

In the new commit, I've separated out `StepProgress` as a new file from `progressResult.yaml` so that both `progressResult` and `operationProgress` can reuse `StepProgress`.

Reply:
I take a closer look, we actually have a bug in the original  `progressResult` yaml, its `progress` field should be list of `operationProgress`.